.gitignore

```diff
@@ -91,3 +91,5 @@ clair-scanner-logs
 release-*
 deploy/olm-certified
 *junit.xml
+# Bias Language Linter
+.biaslanguage/
```

Comment:
Added local script to detect bias language. This ensures linter's code doesn't get into the operator code.